pom.xml

```diff
@@ -29,6 +29,14 @@
         <resource-server.version>1.0.42</resource-server.version>
         <slf4jVersion>1.7.5</slf4jVersion>
         <spring.version>3.2.12.RELEASE</spring.version>
+        <spring-security.version>3.2.5.RELEASE</spring-security.version>
+        <dozer.version>5.5.1</dozer.version>
+        <groovy.version>2.2.2</groovy.version>
+
+        <!-- The JDBC Driver used by the portlet -->
+        <jdbc.groupId>org.hsqldb</jdbc.groupId>
```

Comment:
IMHO I think it is better to have jdbc.* property names for the DB driver you are using (and default to HSQL) and perhaps separately define a run-time dependency on hsql.  Those who fork the project can then run it locally or specify a non-HSQL DB and it would work fine.  I don't think it is a problem having hsql in the deployed WEB-INF/lib as it wouldn't be used if you specified non-HSQL for the JDBC driver.  Also if you are using the Maven Overlay approach, as uPortal does, the overlay process removes hsql and substitutes in whatever is specified in uportal-db/pom.xml anyway.  So I think it's the best of both worlds.

Note an alternative is we switch to using JNDI for DB connections which I'm in favor of if we get group consensus.  Then it's just configuring the multitude of connection pools in your Tomcat server.xml, which we can document.  It brings us one step closer to not having to build with an environment flag and have the environment-specific items be run-time rather than build-time.